pkg/backup/show.go

```diff
@@ -1214,9 +1182,7 @@ var backupShowerDoctor = backupShower{
 	},
 }
 
-func backupShowerFileSetup(
-	p sql.PlanHookState, inCol tree.StringOrPlaceholderOptList,
-) backupShower {
+func backupShowerFileSetup() backupShower {
```

Comment:
This 100% says backup shower (as in taking a shower) and I'm honestly not interested in any argument that says otherwise.

Reply:
indeed